### DISCUSSION TITLE
Switched from Vim and i love it // The importence of having good defaults

### DISCUSSION THREAD

**Discussion by leo-unglaub** [helix-editor/helix#11575] [General] [answered]
Hello dear Helix Community,
i am new to Helix and i have to say, i love it! I have been using Vim/Neovim for the last 17 years non-stop and as good as those editors are, there was always the one thing in the back of my head that i have to tweak this config file or update that plugin. It never had this feeling of just opening a file and typing.

My day job is to maintain around 300 server and routers all over the world. While most of the work is automated, we all know that from time to time we have to ssh onto a machine and fix stuff. Now with my Vim/Neovim config i always had the problem that i had to spend the first 3-5 minutes of fixing stuff on a server, i had to migrate my new config files to the server, run the plugin manager update and then restart Vim/Neovim and then finally i was able to start working. (keep in mind, a normal Neovim installation with a couple of plugin can easily have 300-500 lua files)

**But thanks to the very good defaults in Helix, i am able to work fast with the default installation. Not having to change a single line of config on around 300 servers is such a huge win, i cannot even start telling you how happy this makes me!**

I want to thank the creator and the maintainers of Helix for there work, its an amazing project and i cannot wait for future releases. **I hope that the core stays so strong, that for most of the normal editor tasks, no plugins are needed**. Because just typing `hx` and starting to work is amazing! 

Greetings and thanks!
Leo

**Comment by TornaxO7** [comment, marked as answer]
Welcome to helix!
![peepoAwesome](https://github.com/user-attachments/assets/3a3273c2-fdd9-4a4c-bcd8-40206d48c54d)
